### ISSUE TITLE
Defect 2 - Critical - Content Strategy Statement Image and PDF

### ISSUE THREAD

**Issue by srancour** [department-of-veterans-affairs/va.gov-cms#19370]
## Description or Additional Context
The Content Strategy Statement is an image with lots of detailed text but a limited alt text of "Diagram of the CMS Knowledge Base Content Principles Statement". It also has a downloadable PDF file that consists of a single image that is untagged. This information cannot be reached by the screen reader user.

## Example
The image used on [CMS Knowledge Base content strategy statement](https://prod.cms.va.gov/help/cms-knowledge-base-content-strategy-statement) has limited alt text for how much text is in it. The [content strategy statement PDF](https://prod.cms.va.gov/sites/default/files/2021-07/CMS%20knowledge%20base%20content%20strategy%20principles.pdf) that the user can download isn't tagged with anything and is just a flat image. ![Screenshot of the CMS Knowledge Baser content strategy statement title with the image mentioned and the download link to the PDF](https://github.com/user-attachments/assets/44cca643-6b57-412f-a35d-aec8ea6b71d5)

### Code snippet
Image `<img loading="lazy" src="/sites/default/files/styles/large/public/2021-07/CMS%20knowledge%20base%20content%20strategy%20principles.jpg" width="480" height="315" alt="Diagram of the cms knowledge base content strategy principles" typeof="foaf:Image" class="image-style-large" naptha_cursor="text">`
Link to PDF `<a class="downloadable-file-link downloadable-file-link--document" target="_blank" href="https://prod.cms.va.gov/sites/default/files/2021-07/CMS%20knowledge%20base%20content%20strategy%20principles.pdf" download="https://prod.cms.va.gov/sites/default/files/2021-07/CMS%20knowledge%20base%20content%20strategy%20principles.pdf" aria-label="Download">Download CMS knowledge base content strategy statement  (PDF)</a>`

### Steps to reproduce
Navigate: Home -> Knowledge Base -> How to find Content to Edit -> CMS Knowledge Base content strategy principle
Image is under first heading. Press download link under image to get PDF.

## Recommended resolution
Since all of the text in the image is below the image in plain text, the alt text of the image is a non-issue.
The PDF being inaccessible is an issue and the PDF should be either remade to not just be a flat image and have proper tagging or it should be removed.

All PDFs and images in the KB should be checked to make sure that the alt text makes sense and that the PDFs are accessible.

## Defect Criterion
[WCAG 1.1.1 - Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content) - All [non-text content](https://www.w3.org/TR/WCAG21/#dfn-non-text-content) that is presented to the user has a [text alternative](https://www.w3.org/TR/WCAG21/#dfn-text-alternative) that serves the equivalent purpose, except for the situations listed below.

### Users affected
- Screen reader

## Teams that this affects:
- [x] CMS Team
- [x] sitewide

**Comment by srancour** [comment]
Both subtasks done
